applications/crossbar/priv/couchdb/schemas/callflows.record_call.json

```diff
@@ -4,55 +4,70 @@
+            "description": "Whether to start or stop the recording",
             "enum": [
                 "start",
                 "stop"
             ],
             "name": "Action",
-            "required": true,
+            "required": false,
             "type": "string"
         },
         "format": {
-            "description": "Disk Format to write the recording",
+            "description": "What format to store the recording on disk",
             "enum": [
                 "mp3",
                 "wav"
             ],
             "name": "Format",
-            "required": false,
             "type": "string"
         },
         "record_min_sec": {
-            "description": "Minimal record time, in seconds, to store recordings",
-            "minimum": 0,
-            "name": "Recording minmum seconds",
+            "description": "The minimum length, in seconds, the recording must be to be considered successful. Otherwise it is deleted",
+            "name": "Minimum recording seconds",
             "required": false,
             "type": "integer"
         },
         "record_on_answer": {
-            "description": "Whether to delay starting the recording until the call is answered",
-            "name": "Record on Answer",
+            "default": false,
+            "description": "Whether to delay the recording until the channel is answered",
+            "name": "Record On Answer",
+            "required": false,
+            "type": "boolean"
+        },
+        "record_on_bridge": {
+            "default": false,
+            "description": "Whether to delay the recording until the channel is bridged",
+            "name": "Record On Bridge",
             "required": false,
             "type": "boolean"
         },
         "record_sample_rate": {
-            "description": "Sampling rate of the recording, in Hz",
+            "description": "What sampling rate to use on the recording",
             "name": "Sample Rate",
             "required": false,
             "type": "integer"
         },
+        "recording_id": {
+            "description": "Name the media file that will be generated from the call",
+            "name": "Recording ID",
+            "required": false,
+            "type": "string"
+        },
         "time_limit": {
-            "description": "Limit, in seconds, of how long to record the call",
-            "minimum": 1,
+            "default": 3600,
+            "description": "Time limit, in seconds, for the recording",
+            "maximum": 3600,
+            "minimum": 5,
             "name": "Time Limit",
             "required": false,
             "type": "integer"
         },
         "url": {
-            "description": "What URL to use as a base for where to send the recording after it finishes",
+            "description": "The URL to use when sending the recording for storage",
             "name": "URL",
-            "required": false,
+            "required": true,
```

Comment:
this is not required, if provided will save to url, if not saves locally. i would actually like to remove this and have it set at the data manager level.